elegant-emacs-common.el

```diff
@@ -70,7 +70,7 @@
 
 ;;; When we set a face, we take care of removing any previous settings
 ;;; -------------------------------------------------------------------
-(defun set-face (face style)
+(defun elegant-emacs-common-set-face (face style)
```

Comment:
If we rename the file `elegant.el`, then we could use the `elegant-` prefix everywhere, what do you think ?

Reply:
Yea, could be done if it's ok with you. Where did you want to add the prefix?